docs/cookbook/adding-new-attribute-to-an-entity.md

```diff
@@ -343,4 +343,4 @@ $form['product_edit_form[productData][extId]'] = 123456;
 ```
 
 ## Data fixtures
-Currently, it is not possible to modify data fixtures from your project, this issue will be addressed in near future. 
\ No newline at end of file
+Currently, it is not possible to modify data fixtures from your project, this issue will be addressed in near future. 
```

Comment:
What happened here? I do not see any change :smile: empty line or something like that?

Reply:
There wasn't new line in the end of file, and phpstorm created it. I'll change it to original, so this file is not changed in this commit